README.md

```diff
@@ -1,6 +1,83 @@
+
+
+<a href="h" target="blank"><img align="center" src="https://raw.githubusercontent.com/tigthor/to-do-app/main/ezgif.com-gif-maker%20(2).gif" alt="tigthor" height="300" width="1200" /></a>
+
+
+# Visit Africa  :airplane: :earth_africa:
+Visit Africa Application is the application that will help users to visit Afirca by booking tickets to differents cities getting and paying for accomodations,rating their visits,chatting between users and work with their history of visits and Most importantly Visit Africa applicaion will promote african tourism.
+
+### Technologies :balance_scale: :computer:
+* [Es6+ Javascript](https://www.ecma-international.org/ecma-262/9.0/index.html) The project was written with ES6 syntax and above.
+* [Node/Express](https://nodejs.org/en/) A JavaScript runtime built on Chrome's V8 JavaScript engine.
+* [PostgreSQL](https://www.postgresql.org/) The World's Most Advanced Open Source Relational Database.
+* [Swagger](https://swagger.io/) Used for API documentation. Simplify API development for developers by helping to document APIs.
+* [Pivotal Tracker](https://www.pivotaltracker.com) A project management tool used to manage the app.
+* [Figma](https://www.figma.com/) Used for design mockups. It helps teams to design, prototype, and collaborate all in the browser.
+* [Npm](https://www.npmjs.com/) Used as the package manager for the app. A fast, reliable, and secure dependency management system.
+
+
+
+### UI Design :rocket: :rocket:
+The mockups for the app design can be viewed [here](https://www.pivotaltracker.com/n/projects/2474529)
+
+
+### REST API Docs 	:clap: 	:clap:
+The Api documentation is done using postman. View [VISIT AFRICA Documentation](https://)
+
+### JWT Authentication :raised_hands: :raised_hands:
+The API endpoints are being authenticated using [JWT(Json Web Token)](https://jwt.io/).
+
+### Required Features
+
+```
+
+```
+
+### Installation and Running the Application :collision: :collision:
+
+* Clone the repository using: `git clone https://github.com/Soma-Technologies-Inc/visitAfrica-backend.git`
+* Run `npm install` To install the project dependencies
+* Run `npm run statDev` To start the application
+* Run `npm test` to run test
+
+## Hosted Application can be accessed here 	:fire: :fire: 
+[Hosted on heroku](https://visitafrica-backend.herokuapp.com/)
+
+## Contributors :star_struck: :star_struck: :heavy_heart_exclamation:
+* [Tigthor](https://github.com/tigthor) 
+* [Louange Muhawenubutatu](https://github.com/LouangeMu)
+* [Mutabazi Elysee](https://github.com/engmutabazi100)
+* [Mutoni Hyacinthe](https://github.com/Hyacinthe396)
+* [Benie Mugwaneza Sereine](https://github.com/BenieSereine)
+* [Lilian Nishimwe](https://github.com/Liliane003)
+* [Eustache Akimana](https://github.com/eustacheAki)
+
+<div class="row">
```

Comment:
put a picture under each one name and make it on a line

Reply:
I am gonna look on that